metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Application cookbook which installs and configures Consul.'
 long_description 'Application cookbook which installs and configures Consul.'
-version '2.3.1'
+version '2.3.2'
```

Comment:
Usually, it's not recommended to bump the cookbook version in the scope of the Pull-Request because it's up to the maintainer how & when it should be bumped and released.